base-template.sh

```diff
@@ -1011,12 +952,21 @@ is_single_repo() {
 #   0 if it should be, 1 otherwise
 #####################################################
 should_run_update() {
-    MESSAGE="$(printf "%s\n%s" "* There is a new Githooks update available: Version $LATEST_VERSION" "Would you like to install it now?")"
-    show_prompt EXECUTE_UPDATE "$MESSAGE" "(Yes, no)" "Y/n" "Yes" "no"
 
-    if [ -z "$EXECUTE_UPDATE" ] || [ "$EXECUTE_UPDATE" = "y" ] || [ "$EXECUTE_UPDATE" = "Y" ]; then
-        return 0
+    if is_update_available; then
+
+        MESSAGE="$(printf "%s\n%s" \
+            "* There is a new Githooks update available: Forward-merge to commit \"$(echo "$UPDATE_COMMIT" | cut -c1-7)\"" \
+            "Would you like to install it now?")"
+        show_prompt EXECUTE_UPDATE "$MESSAGE" "(Yes, no)" "Y/n" "Yes" "no"
+
+        if [ -z "$EXECUTE_UPDATE" ] || [ "$EXECUTE_UPDATE" = "y" ] || [ "$EXECUTE_UPDATE" = "Y" ]; then
+            return 0
```

Comment:
So at this point  we have already updated the release checkout.
What's this prompt for? If you say no, we won't revert, is that right?

Reply:
Oh, I see this would be followed by running the new install script from the checkout.
So today this works, when we switch to wrappers and such, we wont.
Can't we do an update check with `git fetch` or something, to see if the remote has a newer commit?

Reply:
And I just saw that this is exactly what you do. 🤦‍♀️
Sorry, misread it.

Reply:
OK, I found the `fetch` but where do we actually pull/merge those changes when we say yes?